docs/walkthrough-1.6.md

```diff
@@ -381,7 +204,7 @@ status:
     type: Ready
 ```
 
-## Step 9 - ServiceInstanceCredential to the ServiceInstance
+# Step 5 - ServiceInstanceCredential to the ServiceInstance
```

Comment:
This reads awkwardly - how about 'Requesting a ServiceInstanceCredential to use the ServiceInstance`?  As-is, doesn't read quite right to me.

Reply:
Good catch - thanks @pmorie

Reply:
Done in #1169